Source/player.cpp

```diff
@@ -730,8 +726,8 @@ bool DoWalk(int pnum, int variant)
 			break;
 		case PM_WALK3:
 			dPlayer[player.position.tile.x][player.position.tile.y] = 0;
-			dFlags[player._pVar4][player._pVar5] &= ~BFLAG_PLAYERLR;
 			player.position.tile = player.position.temp;
+			// Don't delete this (in devilutionX) unncessary assignment. It's needed to stay compatible with vanilla saves.

```

Comment:
```suggestion
			// dPlayer is set here for backwards compatability, without it the player would be invisible is save is loaded in older version
```

Reply:
Please edit `comparability` to `compatibility`